Gemfile

```diff
@@ -85,3 +85,4 @@ group :production do
 end
 
 gem "codeclimate-test-reporter", group: :test, require: nil
+gem "timecop", group: :test
```

Comment:
Remove this after rebase (I added timecop in the development, test group so it will show up after the rebase and we don't want gems twice in the Gemfile).